### DISCUSSION TITLE
Axios dependency prior to 1.6.0 is vulnerable CVE-2023-45857

### DISCUSSION THREAD

**Discussion by JuliusMorkunas** [nrwl/nx#20207] [General]
Hi there,

Are there any plans to update Axios dependency to version 1.6.0+ in the near future?

Axios versions prior to version 1.6.0 have a vulnerability that is described here: https://github.com/axios/axios/issues/6006

Older Axios version used in NX cause security scans to go red in some tools. To prevent that it would be great if NX would update the dependency in the next release.

**Comment by jenseo** [comment]
Agreed, we're right now blocked by our security audits in the pipelines and overriding the dependency gives other issues unfortunately
